cacti/datadog_checks/cacti/cacti.py

```diff
@@ -35,12 +35,13 @@
 }
 
 
-class Cacti(AgentCheck):
+class CactiCheck(AgentCheck):
     def __init__(self, name, init_config, agentConfig, instances=None):
         AgentCheck.__init__(self, name, init_config, agentConfig, instances)
         self.last_ts = {}
 
-    def get_library_versions(self):
+    @classmethod
```

Comment:
This looks more like a use for `@staticmethod` as the method (and the following ones) does not require knowledge about the class itself. I might miss something though, let me know

Reply:
I know, but have been advised to use classmethod instead in past PRs 🤷‍♀

Reply:
I'm pretty sure the advice was then the method was calling itself.

Reply:
What is the difference between self calling and other method in the class calling for this regard?

Reply:
The difference is the scope in which the function itself operates. Eg:

```
@classmethod
def f(cls, val): 
   if stuff:
        # Reference the class
        cls.f(other_val)
```

But in this case:

```
@staticmethod
def g(val):
    # Doesn't operate on anything outside
    return other_val
```

Does that make sense?